stores/appStore.ts

```diff
@@ -125,7 +133,25 @@ export const useAppStore = defineStore("app", {
       });
     },
     clearScenario() {
-      this.currentScenario = { ...emptyScenario };
+      this.currentScenario = structuredClone(emptyScenario);
+    },
+    clearComparison() {
+      this.currentComparison = structuredClone(emptyComparison);
+    },
+    setComparison(axis: string, baselineParameters: ParameterSet, selectedScenarioOptions: string[]) {
+      this.clearComparison();
+      this.currentComparison.axis = axis;
+      this.currentComparison.baseline = baselineParameters[axis];
```

Comment:
You don't want to just re-use `currentScenario` for the baseline stuff? I suppose that might cause problems later on when you can select a different scenario as your baseline, potentially one that you haven't run yet...?
Ah, ok and you're storing all scenario parameter sets in curentComparison.scenarios, including the baseline, and separately keeping a note of what the comparison axis and comparison baseline are.

Reply:
Yeah. I think I was loath to have any crucial details about the comparison living outside of the currentComparison property, and/or I didn't want to give currentScenario two slightly different responsibilities. This feels like a way to reduce chances of currentScenario and currentComparison getting tangled up.